models/backbones/factory.py

```diff
@@ -0,0 +1,21 @@
+from .default_backbone import Backbone
+from .efficientnet import create_efficientnet_backbone
+from .mobilenetv2 import create_mobilenetv2_backbone
+from .resnet import create_resnet_backbone
+
+
+def create_backbone(backbonename: str, alpha: float, weights: str = None):
```

Comment:
please, add docstring like:
def add(a, b):
    """
    Add two numbers and return the result.

    Args:
        a (int): The first number.
        b (int): The second number.

    Returns:
        int: The sum of the two numbers.
    """
    return a + b

or any other docstring format